test/integration/api/ps.bats

```diff
@@ -44,3 +44,61 @@ function teardown() {
 	[ "${#lines[@]}" -eq  2 ]
 	[[ "${lines[1]}" == *"false"* ]]
 }
+
+@test "docker ps --filter" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	# Running
+	firstID=$(docker_swarm run -d --name name1 --label "match=me" --label "second=tag" busybox sleep 10000)
```

Comment:
Does this help?
```
run docker_swarm run -d --name name1 --label "match=me" --label "second=tag" busybox sleep 10000
firstID=$output
[ $status -eq 0 ]
```
Or we just don't care about exit codes of these three runnings?